content/en/docs/started/installing-kubeflow.md

```diff
@@ -5,243 +5,209 @@ weight = 20
 
 +++
 
+## What is Kubeflow?
+
 Kubeflow is an end-to-end Machine Learning (ML) platform for Kubernetes, it provides components for each stage in the ML lifecycle, from exploration through to training and deployment.
 Operators can choose what is best for their users, there is no requirement to deploy every component.
-To read more about the components and architecture of Kubeflow, please see the <a href="/docs/started/architecture/">Kubeflow Architecture</a> page.
 
-### How to install Kubeflow?
+Learn more about Kubeflow in the [Introduction](/docs/started/introduction/) and
+[Architecture](/docs/started/architecture/) pages.
+
+## How to install Kubeflow?
 
-There are two pathways to get up and running with Kubeflow, you may either:
+Anywhere you are running Kubernetes, you should be able to run Kubeflow.
+There are two primary ways to install Kubeflow:
 
-1. [Use a packaged distribution](#packaged-distributions-of-kubeflow)
-2. [Use the raw manifests](#raw-kubeflow-manifests) <sup>(for advanced users)</sup>
+1. [**Packaged Distributions**](#packaged-distributions-of-kubeflow)
+1. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
```

Comment:
```suggestion
2. [**Raw Manifests**](#raw-kubeflow-manifests) <sup>(advanced users)</sup>
```

Reply:
Markdown allows to use the same number twice, and it makes it easier to add new elements without updating multiple lines.

Reply:
And I can't reply to the above comment, so here is a link to my in-thread reply. https://github.com/kubeflow/website/pull/3693/files#r1521923323

Reply:
I see. SGTM